keel-sql/src/main/resources/db/changelog/20200109-create-task-tracking-table.yml

```diff
@@ -0,0 +1,43 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-task-tracking-table
+      author: gyardeni
+      changes:
+        - createTable:
+            tableName: task_tracking
```

Comment:
This table doesn't seem like data that needs to be stored in keel forever, in which case it would be good to add a timestamp field to help with purging old records.

Reply:
@asher I want to add logic that once a task is finished, it'll be removed from this table, so this table will only hold unfinished tasks. Do you think we should add the timestamp in addition?

Reply:
After rethinking this and talking to Emily, will add the timestamp to have the ability to purge or handle records that are not getting handled.